exe/server/main.go

```diff
@@ -106,12 +124,16 @@ func setupInstrumentation() {
 }
 
 func setupLogging() {
-	logging.SetDebugLogging()
-	if !config.GetBool("debug") {
-		logging.SetLogLevel("*", "info")
-		logging.SetLogLevel("rpc", "error")
-		logging.SetLogLevel("dht", "error")
-		logging.SetLogLevel("swarm2", "error")
+	logging.SetLogLevel("*", "error")
+	loggers := []string{"index-miner", "index-ask", "index-slashing", "server",
+		"deals", "fil-toolsd", "fchost", "fpaauth", "fastapi", "ip2location", "reputation"}
```

Comment:
Not convincing yet. Created #141 since this is a static assignment... future modules can be forgot to add here.